### ISSUE TITLE
Empty container-group not allowed in Quarkus 3.8.2

### ISSUE THREAD

**Issue by joggeli34** [quarkusio/quarkus#39316]
### Describe the bug

We like to create containers with a tag/name without the group, eg. `ourregistry.io/api-server'. therefore we used to configure the group to a empty string in the application.properties:
```properties
quarkus.container-image.group=
```

This stopped working with quarkus 3.8.2 with following error message in the gradle build:
```log
* What went wrong:
Execution failed for task ':quarkusAppPartsBuild'.
> SRCFG00040: The config property quarkus.container-image.group is defined as the empty String ("") which the following Converter considered to be null: io.smallrye.config.Converters$BuiltInConverter
```

### Expected behavior

A container is created without the group name

### Actual behavior

The error mention above is fails the gradle build

### How to Reproduce?

Set the property:
```properties
quarkus.container-image.group=
```
and try to build a contaienr

### Output of `java -version`

21

### Quarkus version or git rev

3.8.2

### Build tool (ie. output of `mvnw --version` or `gradlew --version`)

gradlew 8.6

### Additional information

It used to work even in version 3.8.1

**Comment by bropyro** [comment]
Hello guys.
I see the exact same issue with the propetry `quarkus.http.limits.max-body-size`
Looks like this is an general issue in SmallRye

**Comment by gsmet** [comment]
@radcortez I think we will need your help on this one.

**Comment by radcortez** [comment]
I'll have a look.

**Comment by pemistahl** [comment]
I have the same problem with properties being set to the empty string. Worked flawlessly with Quarkus 3.8.1. Please help.

**Comment by gsmet** [comment]
OK, thanks for confirming. Let's get this fixed for 3.8.3 then!
